stable_baselines/agent.py

```diff
@@ -19,24 +20,22 @@ def main(cfg_file, trained_model):
     print("Config parameters = ", json.dumps(params, sort_keys=True, indent=4))
     yaml_file.close()
 
-    time_dep_seed = int((time.time() - int(time.time() - 0.5)) * 1000)
-
     base_path = os.path.dirname(os.path.abspath(__file__))
     model_folder = os.path.join(base_path, params["folders"]["parent_dir"], params["settings"]["game_id"],
                                 params["folders"]["model_name"], "model")
 
     # Settings
     settings = params["settings"]
-    settings["player"] = "P1"
+    settings["role"] = "P1"
 
     # Wrappers Settings
     wrappers_settings = params["wrappers_settings"]
     wrappers_settings["reward_normalization"] = False
 
     # Additional obs key list
-    key_to_add = params["key_to_add"]
+    wrappers_settings["additional_wrappers_list"] = [[RamStatesToChannel, {"ram_states": params["ram_states"]}]]
```

Comment:
Actually more for the arena review but why not just name this 'wrappers' instead of 'additional_wrappers_list'. Consumers don't need to know that there are 'our' and 'additional' wrappers. And the list is just redundant.